### ISSUE TITLE
[BUG] latest release container 3.0.1-ls339 won't start

### ISSUE THREAD

**Issue by mammo0** [linuxserver/docker-swag#522]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Hello,

I have the same problem that @alien75 described in [#514](https://github.com/linuxserver/docker-swag/issues/514#issuecomment-2481080683). The latest SWAG release `3.0.1-ls339` won't start.

The s6 service `init-permissions-config` fails with exit code 1.

_Please ignore the `chmod` `Bad address` errors in the log below. They are already handled by issue #514._

### Expected Behavior

The container starts without issues.

Current workaround: Downgrade the image to version `3.0.1-ls338`

### Steps To Reproduce

Start version `3.0.1-ls339` with the default environment variables as described in the README.

Doing the same with version `3.0.1-ls338` works.

### Environment

```markdown
- OS: QNAP NAS (Kernel 5.10.60-qnap)
- How docker service was installed: QNAP App Container Station
```

### CPU architecture

x86-64

### Docker creation

```bash
As described in the README. Additionally I used the following env variables:

-e PUID=1003 \
-e PGID=100 \
-e VALIDATION=dns \
-e DNSPLUGIN=cloudflare \
-e SUBDOMAINS=wildcard
```
```


### Container logs

```bash
[migrations] started
[migrations] 01-nginx-site-confs-default: skipped
[migrations] done
───────────────────────────────────────
      ██╗     ███████╗██╗ ██████╗
      ██║     ██╔════╝██║██╔═══██╗
      ██║     ███████╗██║██║   ██║
      ██║     ╚════██║██║██║   ██║
      ███████╗███████║██║╚██████╔╝
      ╚══════╝╚══════╝╚═╝ ╚═════╝
   Brought to you by linuxserver.io
───────────────────────────────────────
To support the app dev(s) visit:
Certbot: https://supporters.eff.org/donate/support-work-on-certbot
To support LSIO projects visit:
https://www.linuxserver.io/donate/
───────────────────────────────────────
GID/UID
───────────────────────────────────────
User UID:    1003
User GID:    100
───────────────────────────────────────
Linuxserver.io version: 3.0.1-ls339
Build-date: 2024-11-16T16:36:08+00:00
───────────────────────────────────────
    
using keys found in /config/keys
chmod: changing permissions of '/config/nginx/site-confs': Bad address
chmod: changing permissions of '/config/nginx/site-confs/default.conf': Bad address
chmod: changing permissions of '/config/nginx/site-confs/default.conf.sample': Bad address
chmod: changing permissions of '/config/nginx/ssl.conf': Bad address
chmod: changing permissions of '/config/nginx/proxy-confs': Bad address
chmod: changing permissions of '/config/nginx/proxy-confs/README.md': Bad address
chmod: changing permissions of '/config/nginx/nginx.conf': Bad address
chmod: changing permissions of '/config/nginx/proxy.conf': Bad address
chmod: changing permissions of '/config/nginx/nginx.conf.sample': Bad address
chmod: changing permissions of '/config/nginx/ssl.conf.sample': Bad address
chmod: changing permissions of '/config/nginx/dhparams.pem': Bad address
chmod: changing permissions of '/config/nginx/authelia-location.conf.sample': Bad address
chmod: changing permissions of '/config/nginx/authelia-server.conf.sample': Bad address
chmod: changing permissions of '/config/nginx/ldap-server.conf.sample': Bad address
chmod: changing permissions of '/config/nginx/resolver.conf': Bad address
chmod: changing permissions of '/config/nginx/worker_processes.conf': Bad address
chmod: changing permissions of '/config/nginx/authelia-location.conf': Bad address
chmod: changing permissions of '/config/nginx/authelia-server.conf': Bad address
chmod: changing permissions of '/config/nginx/authentik-location.conf': Bad address
chmod: changing permissions of '/config/nginx/authentik-server.conf': Bad address
chmod: changing permissions of '/config/nginx/authentik-location.conf.sample': Bad address
chmod: changing permissions of '/config/nginx/authentik-server.conf.sample': Bad address
chmod: changing permissions of '/config/nginx/ldap-server.conf': Bad address
chmod: changing permissions of '/config/nginx/ldap-location.conf': Bad address
chmod: changing permissions of '/config/nginx/ldap-location.conf.sample': Bad address
chmod: changing permissions of '/config/nginx/proxy.conf.sample': Bad address
chmod: changing permissions of '/etc/logrotate.d/acpid': Bad address
chmod: changing permissions of '/etc/logrotate.d/fail2ban': Bad address
chmod: changing permissions of '/etc/logrotate.d/lerotate': Bad address
chmod: changing permissions of '/etc/logrotate.d/nginx': Bad address
chmod: changing permissions of '/etc/logrotate.d/php-fpm': Bad address
chmod: changing permissions of '/etc/logrotate.d/php-fpm83': Bad address
Variables set:
3
PGID=100
TZ=Europe/Berlin
URL=----
SUBDOMAINS=wildcard
EXTRA_DOMAINS=
ONLY_SUBDOMAINS=false
VALIDATION=dns
CERTPROVIDER=
DNSPLUGIN=cloudflare
EMAIL=----
STAGING=
chmod: changing permissions of '/defaults/etc/letsencrypt/renewal-hooks/deploy': Bad address
chmod: changing permissions of '/defaults/etc/letsencrypt/renewal-hooks/deploy/10-default': Bad address
chmod: changing permissions of '/defaults/etc/letsencrypt/renewal-hooks/post': Bad address
chmod: changing permissions of '/defaults/etc/letsencrypt/renewal-hooks/post/10-nginx': Bad address
chmod: changing permissions of '/defaults/etc/letsencrypt/renewal-hooks/pre': Bad address
chmod: changing permissions of '/defaults/etc/letsencrypt/renewal-hooks/pre/10-nginx': Bad address
Using Let's Encrypt as the cert provider
SUBDOMAINS entered, processing
Wildcard cert for ---- will be requested
E-mail address entered: ----
dns validation via cloudflare plugin is selected
Certificate exists; parameters unchanged; starting nginx
chmod: changing permissions of '/etc/logrotate.d/acpid': Bad address
chmod: changing permissions of '/etc/logrotate.d/fail2ban': Bad address
chmod: changing permissions of '/etc/logrotate.d/lerotate': Bad address
chmod: changing permissions of '/etc/logrotate.d/nginx': Bad address
chmod: changing permissions of '/etc/logrotate.d/php-fpm': Bad address
chmod: changing permissions of '/etc/logrotate.d/php-fpm83': Bad address
chmod: changing permissions of '/config/log/nginx': Bad address
chmod: changing permissions of '/config/log/nginx/error.log': Bad address
chmod: changing permissions of '/config/log/nginx/access.log': Bad address
chmod: changing permissions of '/config/log/nginx/error.log.2.gz': Bad address
chmod: changing permissions of '/config/log/nginx/error.log.3.gz': Bad address
chmod: changing permissions of '/config/log/nginx/access.log.2.gz': Bad address
chmod: changing permissions of '/config/log/nginx/access.log.4.gz': Bad address
chmod: changing permissions of '/config/log/nginx/access.log.3.gz': Bad address
chmod: changing permissions of '/config/log/nginx/access.log.6.gz': Bad address
chmod: changing permissions of '/config/log/nginx/error.log.5.gz': Bad address
chmod: changing permissions of '/config/log/nginx/error.log.4.gz': Bad address
chmod: changing permissions of '/config/log/nginx/access.log.5.gz': Bad address
chmod: changing permissions of '/config/log/nginx/error.log.7.gz': Bad address
chmod: changing permissions of '/config/log/nginx/error.log.6.gz': Bad address
chmod: changing permissions of '/config/log/nginx/error.log.8.gz': Bad address
chmod: changing permissions of '/config/log/nginx/access.log.7.gz': Bad address
chmod: changing permissions of '/config/log/nginx/access.log.9.gz': Bad address
chmod: changing permissions of '/config/log/nginx/access.log.8.gz': Bad address
chmod: changing permissions of '/config/log/nginx/error.log.9.gz': Bad address
chmod: changing permissions of '/config/log/nginx/error.log.10.gz': Bad address
chmod: changing permissions of '/config/log/nginx/error.log.11.gz': Bad address
chmod: changing permissions of '/config/log/nginx/access.log.10.gz': Bad address
chmod: changing permissions of '/config/log/nginx/access.log.12.gz': Bad address
chmod: changing permissions of '/config/log/nginx/access.log.11.gz': Bad address
chmod: changing permissions of '/config/log/nginx/access.log.14.gz': Bad address
chmod: changing permissions of '/config/log/nginx/error.log.13.gz': Bad address
chmod: changing permissions of '/config/log/nginx/error.log.12.gz': Bad address
chmod: changing permissions of '/config/log/nginx/access.log.13.gz': Bad address
chmod: changing permissions of '/config/log/nginx/error.log.14.gz': Bad address
chmod: changing permissions of '/config/log/nginx/unauthorized.log': Bad address
chmod: changing permissions of '/config/log/nginx/access.log.1': Bad address
chmod: changing permissions of '/config/log/nginx/error.log.1': Bad address
chmod: changing permissions of '/config/log/php': Bad address
chmod: changing permissions of '/config/log/php/error.log': Bad address
chmod: changing permissions of '/config/log/php/error.log.1': Bad address
chmod: changing permissions of '/config/log/php/error.log.2.gz': Bad address
chmod: changing permissions of '/config/log/php/error.log.3.gz': Bad address
chmod: changing permissions of '/config/log/php/error.log.4.gz': Bad address
chmod: changing permissions of '/config/log/php/error.log.5.gz': Bad address
chmod: changing permissions of '/config/log/php/error.log.6.gz': Bad address
chmod: changing permissions of '/config/log/php/error.log.7.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.2.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.3.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.4.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.5.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.6.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.7.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.8.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.9.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.1': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.10.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.11.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.12.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.13.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.14.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.15.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.16.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.17.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.18.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.19.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.20.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.21.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.22.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.23.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.24.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.25.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.26.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.27.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.28.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.29.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.30.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.31.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.32.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.33.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.34.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.35.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.36.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.37.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.38.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.39.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.40.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.41.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.42.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.43.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.44.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.45.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.46.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.47.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.48.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.49.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.50.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.51.gz': Bad address
chmod: changing permissions of '/config/log/letsencrypt/letsencrypt.log.52.gz': Bad address
chmod: changing permissions of '/config/log/fail2ban': Bad address
chmod: changing permissions of '/config/log/fail2ban/fail2ban.log': Bad address
chmod: changing permissions of '/config/log/fail2ban/fail2ban.log.1': Bad address
chmod: changing permissions of '/config/log/fail2ban/fail2ban.log.2.gz': Bad address
chmod: changing permissions of '/config/log/fail2ban/fail2ban.log.3.gz': Bad address
chmod: changing permissions of '/config/log/fail2ban/fail2ban.log.4.gz': Bad address
chmod: changing permissions of '/config/log/fail2ban/fail2ban.log.5.gz': Bad address
chmod: changing permissions of '/config/log/fail2ban/fail2ban.log.6.gz': Bad address
chmod: changing permissions of '/config/log/fail2ban/fail2ban.log.7.gz': Bad address
chmod: changing permissions of '/config/log/logrotate.status': Bad address
s6-rc: warning: unable to start service init-permissions-config: command exited 1
/run/s6/basedir/scripts/rc.init: warning: s6-rc failed to properly bring all the services up! Check your logs (in /run/uncaught-logs/current if you have in-container logging) for more information.
/run/s6/basedir/scripts/rc.init: fatal: stopping the container.
```

**Comment by alien75** [comment]
It it possible to refer to a previous image in compose file?

**Comment by mammo0** [comment]
It's the same as on the CLI: Just add a colon after the image name:
```
services:
  swag:
    image: lscr.io/linuxserver/swag:3.0.1-ls338
```

**Comment by alien75** [comment]
It works perfectly, thank you!

**Comment by mammo0** [comment]
@thespad sorry I don't have a Discord account

**Comment by thespad** [comment]
TL;DR we made a change yesterday that inadvertently turned the existing QNAP issue into a breaking one.

**Comment by mammo0** [comment]
Ok, then I will stay on the latest working version.
